synapse/app/homeserver.py

```diff
@@ -357,6 +357,13 @@ def setup(config_options: List[str]) -> SynapseHomeServer:
         # generating config files and shouldn't try to continue.
         sys.exit(0)
 
+    if config.worker:
```

Comment:
I would imagine this is always truthy? Since I think `config.worker` is a `WorkerConfig`? 

```suggestion
    if config.worker.worker_app:
```

Reply:
Yes I discovered this 5 minutes before the end of the day and beginning of vacation last week :)